commands/qemu-guest-tools/actions_test.go

```diff
@@ -0,0 +1,261 @@
+package qemuguesttools
+
+import (
+	"bytes"
+	"crypto/rand"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http/httptest"
+	"net/url"
+	"os"
+	"path/filepath"
+	goruntime "runtime"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/taskcluster/taskcluster-worker/engines"
+	"github.com/taskcluster/taskcluster-worker/engines/qemu/metaservice"
+	"github.com/taskcluster/taskcluster-worker/runtime"
+)
+
+func fmtPanic(a ...interface{}) {
+	panic(fmt.Sprintln(a...))
+}
+
+func nilOrPanic(err error, a ...interface{}) {
+	if err != nil {
+		fmtPanic(append(a, err)...)
+	}
+}
+
+func assert(condition bool, a ...interface{}) {
+	if !condition {
+		fmtPanic(a...)
+	}
+}
```

Comment:
I'm too lazy to carry around a reference to the `t` object... I just panic instead - so simple :)